packages/@okta/vuepress-site/docs/guides/mobile-swift-configure-redirect/main/index.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Configure Swift SDK redirect authentication
+excerpt: Control if information, such as remembering the device is passed between the web authentication session and your app.
+layout: Guides
+---
+
+Control if information, such as remembering the device is passed between the web authentication session and your app.
```

Comment:
Thanks... I reworked this a bit and came up with:
Control access to the storage used by the redirect authentication session, such as cookies that are used to remember the device.

It will be in the next commit.